pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -509,117 +513,68 @@ class Measurements final
      */
     auto probs(const std::vector<std::size_t> &wires)
         -> std::vector<PrecisionT> {
-        PL_ABORT_IF_NOT(
-            std::is_sorted(wires.cbegin(), wires.cend()),
-            "LightningKokkos does not currently support out-of-order wire "
-            "indices with probability calculations");
-
-        // If all wires are requested, dispatch to `this->probs()`
-        if (wires.size() == this->_statevector.getNumQubits()) {
-            return this->probs();
+        const std::size_t n_wires = wires.size();
+        if (n_wires == 0) {
+            return {1.0};
```

Comment:
Do we want to ensure the precision of  1.0 is aligned with `PrecisionT`?

Reply:
I think this invokes the return constructor via list initialization, which should handle `PrecisionT`, no?